src/plots/gl3d/scene.js

```diff
@@ -219,6 +219,12 @@ proto.initializeGLPlot = function() {
         if(scene.isAspectChanged(layout)) {
             // scene updates
             update[scene.id + '.aspectratio'] = scene.glplot.getAspectratio();
+
+            if(layout[scene.id].aspectmode !== 'manual') {
```

Comment:
How do we get back to the original `aspectmode` ?

Reply:
Should we?

Reply:
When resetting the view (e.g via the modebar buttons), yes.

Reply:
Good call. I think we have some tests related to that.

Reply:
https://github.com/plotly/plotly.js/blob/4d24c36c93d1d1cca4ad4a195cbaf43f53e963fe/test/jasmine/tests/plot_api_test.js#L394-L411

Reply:
So, to be clear, once a user scrolls into a orthographic scene, the `aspectmode` is turned to `'manual'` and the only way to get back to original `aspectmode` is by resetting the view?

Reply:
Or we should keep that in `viewInitial` props?

Reply:
How is it handled currently?

Reply:
`apsectmode` added to `viewInitial` in https://github.com/plotly/plotly.js/pull/4579/commits/1fdc49c7c22dc3b73f7a0f3572b675ea2457b5b7 and tests are added in https://github.com/plotly/plotly.js/pull/4579/commits/6ae280967e5e2e91762d28cc77290516735efc83.